### DISCUSSION TITLE
Cask

### DISCUSSION THREAD

**Discussion by Hs2pMdS6n8** [Homebrew/discussions#4459] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.15
ORIGIN: https://github.com/Homebrew/brew
HEAD: 763c41f006ab9b44261e313b0aaa24a047431b06
Last commit: 3 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 4e69e03a9e49ad29ae957ff00121b85ba38bb8b2
Core tap last commit: 10 weeks ago
Core tap branch: master
Core tap JSON: 27 Apr 13:12 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 13.3.1-arm64
CLT: 14.3.0.0.1.1679647830
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  sdl_net
  sdl_sound
```

### Description of issue

`brew services restart jackett` is creating a LaunchAgent on my machine that isn't working.

In short...
The above creates a LaunchAgent containing the below
```
<key>WorkingDirectory</key>
	<string>/usr/local/Cellar/jackett/0.20.3996/libexec</string>
```

For whatever reason, Jackett just stopped running automatically. Best I can tell some upgrade to Jackett did this (no recent macOS upgrades)

If I change it to this, everything works perfectly.

```
<key>WorkingDirectory</key>
	<string>/opt/homebrew/Cellar/jackett/0.20.3996/libexec</string>
```

I also tested what happens if I download Jackett directly from GitHub (avoiding Homebrew), dumping the `install_service_macos` file into the above /opt/homebrew... location.

It creates a LaunchAgent with the correct path.

That's about all I know. Drove me insane enough I figured it was worth reporting.

### Relevant casks

jackett

**Comment by ZhongRuoyu** [comment]
Does the following work for you:

```
brew upgrade jackett
brew services restart jackett
```

Or if not, can you paste the contents in `/opt/homebrew/opt/jackett/homebrew.mxcl.jackett.plist` here?

**Comment by Hs2pMdS6n8** [comment]
Nope, that broke things again.
Here's the plist from that dir, as requested.

[homebrew.mxcl.jackett.plist.txt](https://github.com/Homebrew/discussions/files/11357827/homebrew.mxcl.jackett.plist.txt)

**Reply by ZhongRuoyu** [reply]
Thanks for reporting this; it should be fixed in https://github.com/Homebrew/homebrew-core/pull/129736. Please do this again:

```
brew reinstall jackett
brew services restart jackett
```

**Comment by Hs2pMdS6n8** [comment]
Yep, works now. Thanks!
